pkg/api/events.go

```diff
@@ -1,21 +1,6 @@
-/*******************************************************************************
-*
-* Copyright 2022 SAP SE
-*
-* Licensed under the Apache License, Version 2.0 (the "License");
-* you may not use this file except in compliance with the License.
-* You should have received a copy of the License along with this
-* program. If not, you may obtain a copy of the License at
-*
-*     http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
-*
-*******************************************************************************/
+// Copyright 2022 - 2025 SAP SE
```

Comment:
Not exactly sure why we switched to SPDX-FileCopyrightText everywhere in https://github.com/sapcc/go-makefile-maker/pull/268 but could be due to reuse verify.

Reply:
I'm not sure. I followed the instructions as I saw them. I'm happy to take whichever action is required, if you want to give me what the reuse annotate command you guys are using, I will follow it.

reuse annotate seemed to be the correct way to go about it.

Reply:
Let's clarify this with Stefan next week. 

Also we have a command in gomm to forcefully replace existing headers.

Reply:
I was following the lead of Archer re: header format, see the description on the PR that Sandro linked.